src/Core/3DTiles/C3DTBoundingVolume.js

```diff
@@ -1,44 +1,123 @@
+const southEastUpCarto = new Coordinates('EPSG:4326');
+const southEastUpVec3 = new THREE.Vector3();
+const northWestBottomCarto = new Coordinates('EPSG:4326');
+const northWestBottomVec3 = new THREE.Vector3();
+const radiusScratch = new THREE.Vector3();
+
+// Culling scratch value
 const worldCoordinateCenter = new THREE.Vector3();
 
+//
+/**
+ * Bounding region is converted to a bounding sphere to simplify and speed computation and culling. This function
+ * computes a sphere enclosing the bounding region.
+ * @param {Object} region - the parsed json from the tile representing the region
+ * @param {THREE.Matrix4} tileMatrixInverse - the inverse transformation matrix of the tile to transform the produced
+ * sphere from a global to a reference local to the tile
+ * @return {THREE.Sphere} a sphere enclosing the given region
+ */
+function initFromRegion(region, tileMatrixInverse) {
+    const east = region[2];
+    const west = region[0];
+    const south = region[1];
+    const north = region[3];
+    const minHeight = region[4];
+    const maxHeight = region[5];
+
+    const eastDeg = THREE.MathUtils.radToDeg(east);
+    const westDeg = THREE.MathUtils.radToDeg(west);
+    const southDeg = THREE.MathUtils.radToDeg(south);
+    const northDeg = THREE.MathUtils.radToDeg(north);
+
+    northWestBottomCarto.setFromValues(westDeg, northDeg, minHeight);
+    ellipsoid.cartographicToCartesian(northWestBottomCarto, northWestBottomVec3);
+
+    southEastUpCarto.setFromValues(eastDeg, southDeg, maxHeight);
+    ellipsoid.cartographicToCartesian(southEastUpCarto, southEastUpVec3);
+
+    const regionCenter = new THREE.Vector3();
+    regionCenter.lerpVectors(northWestBottomVec3, southEastUpVec3, 0.5);
+    const radius = radiusScratch.subVectors(northWestBottomVec3, southEastUpVec3).length() / 2;
+
+    const sphere = new THREE.Sphere(regionCenter, radius);
+    sphere.applyMatrix4(tileMatrixInverse);
+
+    return sphere;
+}
+
+/**
+ * Create a bounding box from a json describing a box in a 3D Tiles tile.
+ * @param {Object} box - the parsed json from the tile representing the box
+ * @return {THREE.Box3} the bounding box of the tile
+ */
+function initFromBox(box) {
+    // box[0], box[1], box[2] = center of the box
+    // box[3], box[4], box[5] = x axis direction and half-length
+    // box[6], box[7], box[8] = y axis direction and half-length
+    // box[9], box[10], box[11] = z axis direction and half-length
+    boxCenter.set(box[0], box[1], box[2]);
+    boxSize.set(box[3], box[7], box[11]).multiplyScalar(2);
+    const box3 = new THREE.Box3();
+    box3.setFromCenterAndSize(boxCenter, boxSize);
+    return box3;
+}
+
+/**
+ * Creats a bounding sphere from a json describing a sphere in a 3D Tiles tile.
+ * @param {Object} sphere - the parsed json from the tile representing the sphere
+ * @returns {THREE.Sphere} the bounding sphere of the tile
+ */
+function initFromSphere(sphere) {
+    const sphereCenter = new THREE.Vector3();
```

Comment:
this was the cause of #1853 and #1888 : the same `Vector3` instance was used for all spehres bounding boxes and the center is not cloned in [Sphere constructor](https://github.com/mrdoob/three.js/blob/master/src/math/Sphere.js#L12).
For box it's ok to use the same `boxCenter` and `boxSize` instance since they are used in `setFromCenterAndSize` which are not stored directly in the box.

Reply:
Ok I see. Thanks for the explanation.